### DISCUSSION TITLE
What type of traffic can Zoraxy proxy?

### DISCUSSION THREAD

**Discussion by PassiveLemon** [tobychui/zoraxy#87] [Q&A] [answered]
I haven't really checked but what all can Zoraxy proxy? Could I proxy SSH/other types of traffic or is this only an HTTP(S) proxy?

**Comment by tobychui** [comment, marked as answer]
@PassiveLemon By default http + https, and TCP proxy allows you to proxy anything that communicate on top of TCP/IP from SSH to Minecraft Multiplayer connections. Though, if you want to use TCP proxy, you will need to open & forward the corresponding port(s) on your docker container or otherwise it will get blocked by firewall. 
Btw, please kindly tag me next time if you have questions as new discussion post never show up in my notification 🧐
